db/queries.py

```diff
@@ -1,46 +1,72 @@
+from datetime import date
 from db.models import Message, User, Chat
 from django.db.models import Q, Count, F
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return list(Message.objects.filter(text__icontains=word))
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return list(Chat.objects.filter(title__startswith="Untitled"))
 
 
 def get_users_who_sent_messages_in_2015() -> list[str]:
-    pass
+    message = Message.objects.filter(sent__year=date(2015, 1, 1).year)
+    return list(message.values_list("user__first_name", "user__last_name"))
```

Comment:
Remove redundant variable. Also, you can write `User.objects.filter(message__sent__year=...)...`